### ISSUE TITLE
elementaryos7(22.04.1-Ubuntu) not working

### ISSUE THREAD

**Issue by aching123** [an-anime-team/an-anime-game-launcher#90]
```
mosyn@aor:~$ flatpak run moe.launcher.an-anime-game-launcher 
thread 'main' panicked at 'index out of bounds: the len is 0 but the index is 0', src/ui/first_run/download_components.rs:268:28
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by Mar0xy** [comment]
run this in terminal to temporarily downgrade to 3.0.1 as 3.1.1 broke something very important
`sudo flatpak update --commit=9d486bfdfbd171337397d53691a89b151a9139fdc88782d293db394c0b9a800b moe.launcher.an-anime-game-launcher`

**Comment by Mar0xy** [comment]
This bug was fixed in 3.1.2 and is currently being pushed to the flatpak which should have the update available in 9-10 mins
